src/main/java/SYSC4806/Controller/HomeController.java

```diff
@@ -1,11 +1,88 @@
 package SYSC4806.Controller;
+
+import SYSC4806.Model.Category;
+import SYSC4806.Model.Course;
+import SYSC4806.Model.LearningOutcome;
+import SYSC4806.Model.Program;
+import SYSC4806.Repository.CategoryRepository;
+import SYSC4806.Repository.CourseRepository;
+import SYSC4806.Repository.LearningOutcomeRepository;
+import SYSC4806.Repository.ProgramRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
 
 @Controller
 public class HomeController {
+
+    @Autowired
+    private CategoryRepository categoryRepository;
+
+    @Autowired
+    private CourseRepository courseRepository;
+
+    @Autowired
+    private LearningOutcomeRepository learningOutcomeRepository;
+
+    @Autowired
+    private ProgramRepository programRepository;
+
     @GetMapping("/")
-    public String home() {
+    public String home(Model model) {
+        model.addAttribute("categories", categoryRepository.findAll());
+        model.addAttribute("courses", courseRepository.findAll());
+        model.addAttribute("learningoutcomes", learningOutcomeRepository.findAll());
+        model.addAttribute("programs", programRepository.findAll());
         return "index";
     }
+
+    //    Adding and Removing data for admin (milestone 2?)
```

Comment:
Since this is for admin, can you remove this from the commit? I just don't want it pushed to master since it's incomplete.